tango_simlib/sim_xmi_parser.py

```diff
@@ -256,6 +318,8 @@ def extract_command_description_data(self, description_data):
         output_parameter = description_data.find('argout')
         command_data['argoutDescription'] = output_parameter.attrib['description']
         command_data['argoutType'] = self._get_arg_type(output_parameter)
+        command_data['inherited'] = (
+                description_data.find('status').attrib['inherited'])
         return command_data
```

Comment:
That's great - so you will be adding a method to prune the inherited items from the tree, or for an external user to get the tree and do it themselves (if it is _get_xmi_tree you should rename it without the underscore). 
May be a method to prune the inherited items from the tree, and a script to re-write the XMI, as I think it may be a useful utility script in tango-simlib that other users may also use if it is available.